CreeDictionary/API/models.py

```diff
@@ -164,6 +161,16 @@ def is_inflection(self) -> bool:
 class Wordform(models.Model):
     _cree_fuzzy_searcher = None
 
+    @property
+    def paradigm(self) -> List[Layout]:
```

Comment:
this was in `views.py`. But I feel like it belongs here more. Since this file is `models.py`. Maybe it's better to not use Layout class here.

Reply:
Honestly, this file is getting WAY too big nowadays; I'll leave that refactoring for a different PR though ;)